templates/_helpers.tpl

```diff
@@ -491,9 +497,54 @@ securityContext for the injector container level.
 {{- define "injector.securityContext.container" -}}
   {{- if .Values.injector.securityContext.container}}
           securityContext:
-            {{- toYaml .Values.injector.securityContext.container | nindent 12 }}
+            {{- $tp := typeOf .Values.injector.securityContext.container }}
+            {{- if eq $tp "string" }}
+              {{- tpl .Values.injector.securityContext.container . | nindent 12 }}
+            {{- else }}
+              {{- toYaml .Values.injector.securityContext.container | nindent 12 }}
+            {{- end }}
   {{- end }}
```

Comment:
Should we move the openshift check into the helper here to match server and csi? i.e.

```suggestion
  {{- else if not .Values.global.openshift }}
```

And remove the check from injector-deployment.yaml.

Reply:
Oh, nevermind -- I see, we should move the default out of `values.yaml` anyway.

Reply:
Done.